changelog/unreleased/changes/1862--bytes-please.md

```diff
@@ -0,0 +1,2 @@
+The memory counts in the output of `vast status` from now on use bytes
+instead of kilobytes for the unit.
```

Comment:
I would label this a bug fix rather than a change, because we used bytes and kilobytes inconsistently in the status output. Additionally, small typo:

```suggestion
The memory counts in the output of `vast status` now represent bytes consistently, as opposed to a mix of bytes and kilobytes. 
```